# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../rt61/eth-naming.patch
# Copyright (C) 2006 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Module/rtmp_main.c~	2000-01-01 00:00:00.000000000 +0100
+++ ./Module/rtmp_main.c	2007-01-02 21:01:15.000000000 +0100
@@ -275,7 +275,7 @@
 	// register_netdev() will call dev_alloc_name() for us
 	// TODO: Remove the following line to keep the default eth%d name
 	if (ifname == NULL)
-		strcpy(net_dev->name, "ra%d");
+		strcpy(net_dev->name, "eth%d");
 	else
 		strncpy(net_dev->name, ifname, IFNAMSIZ);
 
